lib/showoff.rb

```diff
@@ -858,7 +858,6 @@ def update_commandline_code(slide)
           lines = out.split("\n")
           if lines.first.strip[0, 3] == '@@@'
             lang = lines.shift.gsub('@@@', '').strip
-            pre.set_attribute('class', 'highlight')
```

Comment:
CommonMarker generates `<pre><code>`, while showoff generates `<pre class="highlight"><code>`. My suggestion is to remove the highlight class from `<pre>` in the CSS so it works with CommonMarker-generated HTML, and if we do that, then there's no need to add the highlight class anymore.